src/Data/Fin/Mod.agda

```diff
@@ -0,0 +1,111 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- ℤ module n
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Data.Fin.Mod where
+
+open import Function using (id; _$_; _∘_)
+open import Data.Bool using (true; false)
+open import Data.Product
```

Comment:
The three imports above can all be `Base` imports

Reply:
I put as Base.